requirements.txt

```
@@ -1,3 +1,3 @@
-unstract-client~=0.1.0
+unstract-client~=0.2.0
```

@muhammad-ali-e shouldn't this had been 
```suggestion
unstract-client~=1.0.0
```